src/bakery_manager_mobile/lib/pages/edit_account.dart

```diff
@@ -146,7 +137,71 @@ class _EditAccountPageState extends State<EditAccountPage> {
       String username = usernameController.text;
       String password = passwordController.text;
 
-      Map<String, dynamic> response = await Future.delayed(const Duration(seconds: 2), () {
+      // Prepare to delete old emails
+      List<Map<String, dynamic>> emailsToAdd = [];
+
+      for (var email in originalEmails) {
+        await ApiService.deleteUserEmail(emailAddress: email);
+      }
+            // Delete old phone numbers
+      for (var phone in originalPhones) {
+        await ApiService.deleteUserPhone(phoneNumber: phone);
+      }
+
+
+      // Check for new emails to add
+      for (var i = 0; i < _emailControllers.length; i++) {
+        String emailAddress = _emailControllers[i].text;
+        if (emailAddress.isNotEmpty) {
+          emailsToAdd.add({
+            'address': emailAddress,
+            'type': emailTypes[i],
+            'isPrimary': _emails[i]['primary'],
```

Comment:
It does not autoselect the primary radio when you go back to Edit Account. however, I've already made changes on my branch regarding the radio if we want to merge with that first